### PR TITLE
Fix JRuby build on master

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

**Comment by Mogztter** [comment]
@elia bundler issue is gone with this fix but the build is still failing on some tests:


```
Integer#to_f bignum returns self converted to a Float FAILED
Expected 13835069737789293000 to have same value and type as 13835069737789300000
SpecExpectationNotMetError: Expected 13835069737789293000 to have same value and type as 13835069737789300000
```

```
Integer#to_f bignum returns self converted to a Float FAILED
Expected 13835069737789293000 to have same value and type as 13835069737789300000
SpecExpectationNotMetError: Expected 13835069737789293000 to have same value and type as 13835069737789300000
```

**Comment by iliabylich** [comment]
@Mogztter Could you add these 2 specs into `filters/bugs` please?

**Comment by Mogztter** [comment]
@iliabylich Sure, I will do it tomorrow 😀 

> I don't understand why they pass on other rubies.

The value is rounded ? 🤔

File: .travis.yml [hunk 1]
```diff
@@ -70,10 +70,8 @@ matrix:
       env: RUN=rspec
 
     - rvm: jruby-9.2.4.1
-      env: RUN=rspec REINSTALL_BUNDLER=true
```
Comment (elia) [inline .travis.yml]: You should just remove `REINSTALL_BUNDLER=true` here, that's why the failures on jruby popped up. That doesn't explain the error, but explains why we didn't have it before.

File: .travis.yml [hunk 2]
```diff
@@ -70,10 +70,8 @@ matrix:
...
 
     - rvm: jruby-head
-      env: RUN=rspec REINSTALL_BUNDLER=true
```
Comment (elia) [inline .travis.yml]: same as above

File: .travis.yml (hunk 1 above)
Comment (Mogztter) [reply .travis.yml]: Ooops, nice catch :flushed:

File: .travis.yml (hunk 2 above)
Comment (Mogztter) [reply .travis.yml]: fixed

File: .travis.yml (hunk 1 above)
Comment (Mogztter) [reply .travis.yml]: fixed

**Comment by Mogztter** [comment]
Build is green again :smile:

**Pull request merged**